fedimint-core/src/api.rs

```diff
@@ -595,6 +599,40 @@ impl WsClientConnectInfo {
         }
     }
 
+    /// Converts the connect info into a bech32 string for compact display / copying
+    pub fn to_bech32(&self) -> anyhow::Result<String> {
+        let mut data = vec![];
+        data.extend(self.id.0.to_bytes());
+        for url in &self.urls {
+            data.extend(";".as_bytes());
+            data.extend(url.as_str().as_bytes());
+        }
```

Comment:
I'd much prefer having a length-prefixed encoding than relying on a delimiter that may very well be part of a valid URL. 

An alternative encoding:
```
[ id (48 bytes) ] ([ url len (2 bytes) ] [ url bytes (url len bytes) ])+
```